service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -24,4 +25,13 @@ public List<Pipeline> getPipelines() {
     logger.info("Get all Pipelines");
     return pipelinesDao.getPipelines();
   }
+
+  public void validatePipeline(String pipelineId) {
+    logger.info("Validate pipeline");
+
+    Boolean pipelineExists = pipelinesDao.checkPipelineExists(pipelineId);
+    if (Boolean.FALSE.equals(pipelineExists)) {
+      throw new PipelineNotFoundException(String.format("Pipeline %s not found.", pipelineId));
+    }
```

Comment:
```suggestion
    if (!pipelineExists) {
      throw new PipelineNotFoundException(String.format("Pipeline %s not found.", pipelineId));
    }
```
wondering if that is more readable

Reply:
ugh, it totally is more readable, and that's what I had originally, but SonarCloud complained that it wouldn't handle a `null` properly and gave this suggestion instead. 🙃

Reply:
I would trust SonarCloud or have it return false by default

Reply:
good point. I'm trying out having the validatePipeline() method return a (non-nullable) `boolean` rather than `Boolean`. we'll see if that makes SonarCloud happy with `!pipelineExists`...

Reply:
SonarCloud is pleased with our efforts! thanks so much for the suggestion.